### ISSUE TITLE
NA being used as title

### ISSUE THREAD

**Issue by robjhyndman** [robjhyndman/forecast#772]
```r
fit <- stl(USAccDeaths, s.window=11)
autoplot(fit) 
```

**Comment by robjhyndman** [comment]
Also in labels

```r
auto.arima(USAccDeaths) %>% forecast() %>% autoplot()
```

**Comment by mitchelloharawild** [comment]
Resolved in 33416896d8fef6414736b0e4e35d40d530dd3e76
